cds-snc/cppd-medical-report-sails
# New consent form and consent views

# What this does
Introduces the new consent form design and views. Also fleshes out the test suite around consent.

## What's missing
- haven't implemented the "Print proof of consent" link yet - need to talk about placement as I don't think it will fit where it is in the designs (separate PR perhaps?)

## To test
This PR comes with tests, so be sure to check them out: `npm run cypress:open` and select "Run all specs" or just run the `consent.spec.js` and `patient_flow.spec.js` specs that cover this functionality.

- Start a new medical report
- Notice the new consent form
- Submit the consent form

### Medical Professional view
- After completing steps above:
- Login to the Medical Professional section at `/en/medical-professional` using the code from the previous step
- View the consent using the link provided

### Medical Adjudicator view
- After completing the steps to submit a consent:
- Visit the Medical Adjudicator sessions view at `/en/sessions`
- Select the report from the list
- View the consent using the link provided
- Be sure to do this for both positive and negative consent
